ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -28,7 +28,7 @@ resource "azurerm_monitor_metric_alert" "mem_util" {
   frequency           = "PT1M"
   window_size         = "PT5M"
   severity            = var.severity
-  enabled             = var.enabled
+  enabled             = contains(var.disabled_alerts, "mem_util") ? false : true
```

Comment:
No `not` keyword? Boooo.

Reply:
Yeah no can do, HCL is a little limited. I could invert the condition if it makes more sense (`!contains(var.disabled_alerts, "mem_util") ? true : false) or omit the ternary, but I think those are a little less readable.

Reply:
Yeah, the ternary makes you actually look, otherwise a `!` is kind of easy to miss.